<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coursefind</class>
 <widget class="QWidget" name="coursefind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>课程查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stu_course_backButton">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>请输入你要查询的课程号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stu_course_IDlineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="stu_query_courseButton">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>课程名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>授课老师：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>授课时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>授课地点：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>学时：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>学分：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="stu_course_namelineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="stu_teacher_namelineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="stu_course_timelineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="stu_course_addresslineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="stu_course_periodlineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="stu_course_creditlineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
